internal/handlers/common/aggregations/operators/accumulators/sum.go

```diff
@@ -120,7 +150,8 @@ func sumNumbers(vs ...any) any {
 		case float64:
 			hasFloat64 = true
 
-			floatSum = floatSum + v
+			// Set precision to 2048, it is assumed precision used for MongoDB.
```

Comment:
Go uses IEEE-754 64-bit floating-point and bson specification says double is 64-bit IEEE 754-2008 binary floating point.

But arithmetic only upon overflow seems to give different result.